potentials/Al_2020.pinn

```diff
@@ -0,0 +1,1076 @@
+# Development of a general-purpose machine-learning interatomic 
```

Comment:
The first line of bundled potential files should follow the conventions of other bundled potential files and have a DATE:,  CONTRIBUTOR:, CITATION:, and UNITS: tag.